Python/Product/PythonTools/PythonTools/Navigation/CodeWindowManager.cs

```diff
@@ -143,8 +145,8 @@ class CodeWindowManager : IVsCodeWindowManager, IVsCodeWindowEvents {
             var wpfTextView = VsEditorAdaptersFactoryService.GetWpfTextView(vsTextView);
             if (wpfTextView != null) {
                 var factory = ComponentModel.GetService<IEditorOperationsFactoryService>();
-                var editFilter = new EditFilter(wpfTextView, factory.GetEditorOperations(wpfTextView), _serviceProvider);
-                editFilter.AttachKeyboardFilter(vsTextView);
+                EditFilter.GetOrCreate(_serviceProvider, ComponentModel, vsTextView);
+                ReplEditFilter.GetOrCreate(_serviceProvider, ComponentModel, vsTextView);
```

Comment:
The refactoring split EditFilter into two classes, and I left the code to attach them in the original location. It's either not the place to attach edit filters at all (probably true), or it's the right place to attach both. I don't know where the better place would be if one exists.